precompiles/staking/IStaking.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.8.26;
+
+/// @dev The IStaking contract's address.
+address constant ISTAKING_PRECOMPILE_ADDRESS = 0x0000000000000000000000000000000000000066; // 102
+
+/// @dev The IStaking contract's instance.
+IStaking constant ISTAKING_CONTRACT = IStaking(
+    ISTAKING_PRECOMPILE_ADDRESS
+);
+
+/// @notice Bond status for validator
+enum BondStatus {
+    Unspecified,
+    Unbonded,
+    Unbonding,
+    Bonded
+}
+
+/// @notice Validator info
+struct Validator {
+    string operatorAddress;
+    string consensusPubKey;
+    bool jailed;
+    BondStatus bondStatus;
+}
+
+interface IStaking {
+    /// @notice Stake coins to validator
+    /// @param staker Staker address
+    /// @param validator Validator address
+    /// @param amount Coins amount
+    /// @return success Staking success
+    function stake(
+        address staker,
+        string memory validator,
+        uint256 amount
+    ) external returns (bool success);
+
+    /// @notice Unstake coins from validator
+    /// @param staker Staker address
+    /// @param validator Validator address
+    /// @param amount Coins amount
+    /// @return completionTime Time when unstaking is done
+    function unstake(
+        address staker,
+        string memory validator,
+        uint256 amount
+    ) external returns (int64 completionTime);
+
+    /// @notice Transfer coins from validatorSrc to validatorDst
+    /// @param staker Staker address
+    /// @param validatorSrc Validator from address
+    /// @param validatorDst Validator to address
+    /// @param amount Coins amount
+    /// @return completionTime Time when staket transfer is done
+    function transferStake(
```

Comment:
Transfer is commonly understood as transferring the ownership to someone else while here the stakes remain ownership of the staker.

I would rather renaming to `moveStake`